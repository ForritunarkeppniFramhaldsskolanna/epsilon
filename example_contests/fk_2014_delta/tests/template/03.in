sundlaug=Sundlaug Gardabaejar
nafn=Jonas
===
Hallo {sundlaug}! Mer finnst mjog gaman ad synda hja ykkur, og tess vegna aetla eg ad saekja um starf hja ykkur.
{nafn}
===
