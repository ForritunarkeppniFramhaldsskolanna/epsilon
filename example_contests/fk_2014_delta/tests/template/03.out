Hallo Sundlaug Gardabaejar! Mer finnst mjog gaman ad synda hja ykkur, og tess vegna aetla eg ad saekja um starf hja ykkur.
Jonas
